api/src/routing.js

```diff
@@ -120,7 +120,7 @@ app.use(
         defaultSrc: ["'none'"],
         fontSrc: ["'self'"],
         manifestSrc: ["'self'"],
-        connectSrc: ["'self'"],
+        connectSrc: ["'self'", "''https://staging.dev.medicmobile.org'"],
```

Comment:
Is the URL supposed to have two single quotes at the front?

The helmet documentation implies this would work:

```js
connectSrc: ["'self'", 'https://staging.dev.medicmobile.org']
```

I think hardcoding it is fine - as you say, it's already hardcoded in API. I'm thinking a test for this would be good so we don't regress if we change the horti server one day. A unit test to compare the two hardcoded strings would be sufficient. If you felt like it you could add an e2e test which makes sure the page renders correctly which would check for other things as well. It could be a little complicated because you'd need to mock the http request so I'll leave it for you to decide whether it's worth it in this case.